src/dataEntryApp/services/SubjectService.js

```diff
@@ -4,7 +4,7 @@ export default {
   search(params) {
     const _params = {
       page: 0,
-      size: 20,
+      size: 1000,
```

Comment:
I notice that we are calling the search api but not passing it pagination parameters from UI. We are hardcoding page size to be 1000.
There is a dropdown on UI for selecting page size so should not we pass it to API? I think may be we are doing some truncating at client side which should not be needed if we pass the page number and page size params to the api itself.